### PR TITLE
refresh collection detail pages (bug 1103202/1103304)

### PR DESCRIPTION
**Pull request by ngokevin**
- mobile designs adapted from the desktop mocks.
- adapted the operator shelf mocks to collections with background images
- i winged the mega collections as there were no mocks for those. 

screen shot 2015-01-08 at 10 48 28 am
screen shot 2015-01-08 at 10 48 35 am
screen shot 2015-01-08 at 10 49 05 am
screen shot 2015-01-08 at 10 48 56 am

screen shot 2015-01-08 at 10 56 32 am
screen shot 2015-01-08 at 10 56 41 am

### CODE REVIEW

**Comment by ngokevin** [comment]
@spasovski

File: src/dev.html [hunk 1]
```diff
@@ -47,6 +47,7 @@
       <link rel="stylesheet" href="/media/css/feed/feed.styl.css">
       <link rel="stylesheet" href="/media/css/feed/brands.styl.css">
       <link rel="stylesheet" href="/media/css/feed/collections.styl.css">
+      <link rel="stylesheet" href="/media/css/feed/landing.styl.css">
```
Comment (spasovski) [inline src/dev.html]: I guess for all feed landing pages?

File: src/media/css/app-list.styl [hunk 2]
```diff
@@ -42,9 +42,11 @@ $desktop-tile-padding = 20px 50px;
     margin-top: 30px;
 
     li:not(.loadmore) {
-        border-bottom: 1px solid $filters-border;
         padding-top: 15px;
     }
+    li:not(.loadmore):not(:last-child) {
```
Comment (spasovski) [inline src/media/css/app-list.styl]: Good catch.

File: src/media/js/main.js [hunk 3]
```diff
@@ -21,6 +21,7 @@ define(
         'content-ratings',
         'forms',
         'image-deferrer',
```
Comment (spasovski) [inline src/media/js/main.js]: Do we still need this one then?

File: src/templates/_macros/feed_item.html [hunk 4]
```diff
@@ -269,7 +263,7 @@ <h1 class="feed-item-header grouped">{{ group.name }}</h1>
     {{ feed_brand(obj.brand) }}
 
   {% elif obj.item_type == 'collection' %}
-    {{ feed_collection(obj.collection) }}
+    {{ collection(obj.collection) }}
```
Comment (spasovski) [inline src/templates/_macros/feed_item.html]: I agree with nuking the `feed` prefix going forward except for the home page.

**Comment by spasovski** [comment]
Not happy with the visual of the mega collections, it's possible we need to revisit the idea of multiple app listing containers (restyle or get rid of them). I'd like input from @pwalm here as well.

File: src/dev.html (hunk 1 above)
Comment (ngokevin) [inline src/dev.html]: common styles between feed landing pages

File: src/media/js/main.js (hunk 3 above)
Comment (ngokevin) [inline src/media/js/main.js]: yeah. image-deferrer is the lib, mkt-image-deferrer is its application

**Comment by ngokevin** [comment]
I should probably be having mobile look the same as it was before. As in the top of the detail page is what is shown on the Feed.

**Pull request closed without merging**